.travis.yml

```diff
@@ -40,7 +44,9 @@ env:
   global:
     # These env vars are available to every build
     - PYTHONPATH=$PYTHONPATH:../refinery:../refinery/config
-    - DJANGO_SETTINGS_MODULE=config.settings.prod
+    - DJANGO_SETTINGS_MODULE=config.settings.dev
```

Comment:
I was thinking of  something more along the lines of on line 76: 
`python manage.py runserver --insecure --settings=config.settings.dev`
so that the cypress testing is the one off thing and we can still run our units against prod settings

Reply:
FYI, unit tests should be unaffected by Django settings because they should only test small pieces of code in isolation.
Also, `--insecure` is [only needed when DEBUG is False](https://docs.djangoproject.com/en/1.7/ref/contrib/staticfiles/#django-admin-option---insecure).

Reply:
[core runserver](https://docs.djangoproject.com/en/1.11/ref/django-admin/#django-admin-runserver) only has `--noreload`,  `--nothreading`, `--ipv6` and the option to pick a port. 

[staticfiles runserver](https://docs.djangoproject.com/en/1.11/ref/contrib/staticfiles/#cmdoption-runserver-insecure) adds `--insecure`.

But I could just reset that envar before runserver?

Reply:
There is only one Django dev server. It is invoked by the `runserver` subcommand. If DEBUG is set to True in the settings (e.g., in dev.py) you simply don't need to use `--insecure` (that mode is insecure by default).
Btw, the docs you've linked to are for Django 1.11 while we are running 1.7 - there often big differences between versions.